### ISSUE TITLE
It doesn't show icons - MBAir M2

### ISSUE THREAD

**Issue by diegomarino** [exelban/stats#2108]
**Describe the bug**

It just doesn't show icons on the menubar. 

Installed through brew, the app starts normally (I can see the settings window), and I can check from the Activity Monitor that it is consuming single digits of CPU. Also, I can quit it without needing to force it.

Last week I installed it in an older Air (M1) and it worked flawlessly. 

I've deactivated [Ice](https://github.com/jordanbaird/Ice) without success. Also, I have an external 49'' monitor I use as main, converted it to secondary and unplugged it (so only the laptop screen is active), without results. It doesn't show the icons in the menubar in any hardware configuration.

Also, I tried to toggle modules individually and combined without results.

Here's a list of the different errors obtained from the console logs:

```
error	11:43:57.761262+0200	Stats	No error handler for XPC error: <private>
error	11:44:01.833128+0200	Stats	Couldn't read values in CFPrefsPlistSource<0x600003629a70> (Domain: eu.exelban.Stats.widgets, User: kCFPreferencesAnyUser, ByHost: Yes, Container: (null), Contents Need Refresh: Yes): Using kCFPreferencesAnyUser with a container is only allowed for System Containers, detaching from cfprefsd
error	11:44:02.568507+0200	Stats	CGSWindowShmemCreateWithPort failed on port 0
error	11:44:02.699234+0200	Stats	quic_crypto_send [C1.1.2.2:2] [-bc7b1ae5570e7bd1] unable to send CRYPTO frames
error	11:44:02.714397+0200	Stats	nw_connection_copy_connected_local_endpoint_block_invoke [C3] Client called nw_connection_copy_connected_local_endpoint on unconnected nw_connection
error	11:44:02.793868+0200	Stats	CGSWindowShmemCreateWithPort failed on port 0
error	11:47:03.564400+0200	Stats	nw_read_request_report [C1] Receive failed with error "Operation timed out"
error	11:47:03.572293+0200	Stats	nw_endpoint_flow_fillout_data_transfer_snapshot copy_info() returned NULL
```
 
HTH 🙂

**Details:**
 - Device: MacBook Air M2, 15'',  24GB
 - macOS: Sonoma 14.0
 - Application version: 2.11.8

**Comment by cosileone** [comment]
I had a similar problem, my icons were not appearing MBP M2 14". I noticed after a restart they were appearing in ice's always-hidden category, I just dragged them all over to the visible section.

I know you tried disabling ice but perhaps a restart is necessary

**Comment by diegomarino** [comment]
Thanks, @cosileone! I tried all possible combinations: opening, closing, uninstalling, and restarting apps 🤷.

In the end, I manually closed all the apps, and the culprit turned out to be OnlySwitch. Inside its settings, there’s an option to hide menu bar items, but since it’s not listed in the main toggles, I didn’t realize it was there. Also, for some reason it didn't show the divider icon.

See screenshot attached

![CleanShot 2024-09-06 at 11 31 55](https://github.com/user-attachments/assets/3a90dd50-a824-47fa-975e-cd84a4355cc0)
